libs/designsystem/menu/src/menu.component.html

```diff
@@ -28,9 +33,13 @@
   [autoPlacement]="autoPlacement"
   [shift]="shift"
   [ngStyle]="{
-    minWidth: minWidth ? minWidth + 'px' : null
+    minWidth: minWidth ? minWidth + 'px' : null,
   }"
```

Comment:
Consider simplifying:
```suggestion
  [style.min-width]="minWidth ? minWidth + 'px' : null"
```